x/auth/ante.go

```diff
@@ -77,17 +77,14 @@ func NewAnteHandler(am AccountKeeper, fck FeeCollectionKeeper) sdk.AnteHandler {
 		// When simulating, this would just be a 0-length slice.
 		stdSigs := stdTx.GetSignatures()
 		signerAddrs := stdTx.GetSigners()
-
-		// create the list of all sign bytes
-		signBytesList := getSignBytesList(newCtx.ChainID(), stdTx, stdSigs)
 		signerAccs, res := getSignerAccs(newCtx, am, signerAddrs)
 		if !res.IsOK() {
 			return newCtx, res, true
 		}
-		res = validateAccNumAndSequence(ctx, signerAccs, stdSigs)
-		if !res.IsOK() {
-			return newCtx, res, true
-		}
+
+		isGenesis := ctx.BlockHeight() == 0
+		// create the list of all sign bytes
+		signBytesList := getSignBytesList(newCtx.ChainID(), stdTx, signerAccs, isGenesis)
```

Comment:
It would still be nice to return an incorrect-sequence-specific error, but I suppose with this change that becomes difficult to do - not a big deal.